schemas/includes/person.json

```diff
@@ -15,6 +15,10 @@
       "type": "string",
       "description": "Name of the jurisdiction in which the entity is based"
     },
+    "company": {
```

Comment:
@inglesp Did we discuss having relationship_with_company as a field? Thought we did, but not sure we got further. I'm thinking of something like an enum: ['employee', 'director', 'shareholder']